src/app/(nextjs_migration)/components/client/ExposureScanInput.tsx

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+import dynamic from "next/dynamic";
+import { useL10n } from "../../../hooks/l10n";
+
+const ExposureScanInput = () => {
+  const l10n = useL10n();
+
+  return (
+    <input
+      id="scan-email-address"
+      name="email"
+      type="email"
+      placeholder={l10n.getString("exposure-landing-hero-email-placeholder")}
+      required
+    />
+  );
+};
+
+export default dynamic(() => Promise.resolve(ExposureScanInput), {
+  ssr: false,
```

Comment:
Hm, is it required to do both this and `use client`? Or are the subtly different in some way? :)

Reply:
`use client` makes this component a client component, but that does not mean they are client-only. The initial DOM elements are still being rendered server-side. Adding a dynamic export with `ssr: false` ensures that there is no part of the component that’s executed on the server.